docs/Module-Review/MPEs/MH4500 Time Series Analysis.md

```diff
@@ -0,0 +1,34 @@
+---
+id: mh4500-time-series-analysis
+slug: mh4500-time-series-analysis
+sidebar_position: 5
+title: MH4500 Time Series Analysis
+description: Developing models for time series data
+keywords: [time series, analysis, ntu, spms, course, module]
+---
+
+## Course Summary
+
+The course looks into what is a time series and techniques on how to deal with and model and time series data. This is a continuation of [MH3510 Regression Analysis](MH3510%20Regression%20Analysis.md) where we will deal with a more specific type of data which is time series data. The topics covered in this course are:
```

Comment:
this is a broken link or the one below

`- On source page path = /docs/Module-Review/MPEs/mh4500-time-series-analysis:
   -> linking to ../MH3510%20Regression%20Analysis.md (resolved as: /docs/Module-Review/MH3510%20Regression%20Analysis.md)`